opm/simulators/linalg/WellOperators.hpp

```diff
@@ -125,6 +154,10 @@ class WellModelAsLinearOperator : public LinearOperatorExtra<X, Y>
 
 protected:
     const WellModel& wellMod_;
+
+    mutable X x_local_{};
```

Comment:
I think `mutable` variables should always be explained in a comment. (Can be lifted from the BlackoilWellModel class.)

Reply:
alright, lifted the comment.